src/Services/Air/AirParser.js

```diff
@@ -249,7 +249,10 @@ function airPriceRspPricingSolutionXML(obj) {
   pricingSolution['air:AirPricingInfo'] = pricingInfos;
   const resultXml = {};
 
-  ['air:AirSegment', 'air:AirPricingInfo', 'air:FareNote'].forEach((root) => {
+  ['air:AirSegment', 'air:AirPricingInfo', 'air:FareNote', `common_${this.uapi_version}:HostToken`].forEach((root) => {
```

Comment:
Are we sure this change fixes the bug? And why we need check for non existing tag?

Reply:
Some carriers return HostToken, some doesn't. 
If we don't check for non existing tag, following code may generate empty HostToken_XML tag which will break following RQ.

Reply:
@creduo kindly waiting for the tests.

Also should we just skip the solution, or should we each solution bookable in a way?

Reply:
@dchertousov Added tests for this case, 
and I don't follow what do you mean by solution. Is it air:AirPricingSolution?